### DISCUSSION TITLE
Site to site single NIC behind NAT

### DISCUSSION THREAD

**Discussion by huglen** [strongswan/strongswan#1003] [Q&A] [answered]
Please bear with me. I have really tried to experiment, search for answers and read documentation.

**Site to site VPN, one side behind NAT.**
Symptom: VPN GW's can connect and ping each other internal IP. If pinging machines on the remote subnet the packages reach GW but never the machine in local LAN.
Machines in the Site 1 LAN reaches the remote VPN GW but packages are never forwarded through lan.
Machines in Site 2 LAN reaches only VPN GW on Site 2 

**Site 1 - Behind NAT only 1 NIC**
Public ip forward and gw is out of my control, they are automatically setup by cloud service.
**Public IP**: 147.189.192.62 (no physical NIC, traffic forwarded to private NIC)
**Private IP**: 192.168.10.218
**Subnet**: 192.168.10.0/24
**GW**: 192.168.10.1

**Site 2**
**Public IP**: 94.237.93.121
**Private IP**: 10.0.152.2
**Subnet**: 10.0.152.0/24

[ip.txt](https://github.com/strongswan/strongswan/files/8532600/ip.txt)
[routing.txt](https://github.com/strongswan/strongswan/files/8532601/routing.txt)
[iptables.txt](https://github.com/strongswan/strongswan/files/8532602/iptables.txt)
[list-sas.txt](https://github.com/strongswan/strongswan/files/8532603/list-sas.txt)
[list-conns.txt](https://github.com/strongswan/strongswan/files/8532604/list-conns.txt)
[log.txt](https://github.com/strongswan/strongswan/files/8532599/log.txt)

**Comment by tobiasbrunner** [comment, marked as answer]
Please read [this](https://docs.strongswan.org/strongswan-docs/5.9/howtos/forwarding.html).

**Reply by huglen** [reply]
Thanks, I did read that documentation over and over again I tried to follow it to the best of my understanding ;-)
You were probably right on target. I had to setup some nat postrouting rules in iptables to get it working. Only on one end though (the one behind NAT) so i am guessing there's some package filtering or rules preventing it from working automatically like it did on the other end.

Thanks for your valuable input.
